### ISSUE TITLE
[Feature] - 

### ISSUE THREAD

**Issue by straussi** [AlpsBTE/Plot-System#79]
Hi, I am Straussi, as Staff at BTE Germany I am responsible to create new and review built plots.
**Is your feature request related to a problem? Please describe**
When submitted and reviewed plots get pasted into the Terra world, noone knows anymore, who built this plot, and which ID it had.
**Describe the solution you'd like**
Therefore it would be a great feature if there was a sign with the ID and the name of the plot builder inside the building, so you know lateron, who built a certain plot.
You could do this by forcing a player to build a sign inside the house, before he can submit the building, the same way you have to place a sign on the street side, when creating a plot.

**Comment by LordTuxn** [comment]
Good idea, shouldn't be much work. Maybe transfer this issue to the Plot-System-Terra repo https://github.com/AlpsBTE/Plot-System-Terra, as we have to implement it there.
